### PR TITLE
Chore/remove deps

### PR DESCRIPTION
**Pull request by 8ball030**
Dependency on submodule removed. 

Readmes updated with the raw docker commands.

Small changes on version bumps for develop image.

### CODE REVIEW

File: develop-image/Dockerfile [hunk 1]
```diff
@@ -44,25 +43,22 @@ RUN apt-get install -y                                                       \
...
 # needed by Pipenv
-ENV DEBIAN_FRONTEND noninteractive
 ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
-RUN apt-get install -y tox
-RUN python3.7 -m pip install -U pipenv
-
-ENV PATH="/usr/local/bin:${PATH}"
-USER default
```
Comment (DavidMinarsch) [inline develop-image/Dockerfile:55]: Not needed?

File: deploy-image/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-alpine
+FROM python:3.7-alpine
```
Comment (DavidMinarsch) [inline deploy-image/Dockerfile:1]: Intentional?

Comment (8ball030) [reply deploy-image/Dockerfile:1]: Yes, setuptools had an issue with the previous version. (see i ensure setuptools is up todate before the build command as well.
./build.sh was failing for version 3.8-alpine

File: develop-image/Dockerfile (hunk 1 above)
Comment (8ball030) [reply develop-image/Dockerfile:55]: I dont think so, 
everything seems to work in the image as I would expected.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**